include/tvm/node/container.h

```diff
@@ -96,8 +98,7 @@ class MapNode : public Object {
  * \tparam V The value NodeRef type.
  */
 template <typename K, typename V,
-          typename = typename std::enable_if<std::is_base_of<ObjectRef, K>::value ||
-                                             std::is_base_of<std::string, K>::value>::type,
+          typename = typename std::enable_if<std::is_base_of<ObjectRef, K>::value>::type,
```

Comment:
I think that it is better to directly change it to static_assert later,

Reply:
Yeah, but the difference is less fundamental imo